package.json

```diff
@@ -20,39 +20,37 @@
     "prepublish": "run-s clean build"
   },
   "devDependencies": {
-    "@aws-cdk/assert": "0.31.0",
-    "@aws-cdk/assets": "0.31.0",
-    "@aws-cdk/assets-docker": "0.31.0",
-    "@aws-cdk/aws-applicationautoscaling": "0.31.0",
-    "@aws-cdk/aws-autoscaling": "0.31.0",
-    "@aws-cdk/aws-autoscaling-api": "0.31.0",
-    "@aws-cdk/aws-autoscaling-common": "0.31.0",
-    "@aws-cdk/aws-certificatemanager": "0.31.0",
-    "@aws-cdk/aws-cloudformation": "0.31.0",
-    "@aws-cdk/aws-cloudwatch": "0.31.0",
-    "@aws-cdk/aws-codedeploy-api": "0.31.0",
-    "@aws-cdk/aws-codepipeline-api": "0.28.0",
```

Comment:
deprecated.
https://www.npmjs.com/package/@aws-cdk/aws-codepipeline-api
(We might depend on `@aws-cdk/aws-codepipeline-actions` instead of this from author message, but I just removed since yarn never complains about its absence.)